client/core/core.go

```diff
@@ -1171,6 +1171,7 @@ func New(cfg *Config) (*Core, error) {
 			return nil, fmt.Errorf("Error parsing language tag %q: %w", cfg.Language, err)
```

Comment:
I'm noticing that `Core's language matching is much more strict that `Webserver`'s.  I think we'll need to mirror the logic used in `Webserver`'s constructor here.  Something along the lines of this:

```go
		acceptLangs, _, err := language.ParseAcceptLanguage(cfg.Language)
		if err != nil {
			return nil, fmt.Errorf("unable to parse requested language: %w", err)
		}
		// fmt.Println("parsed input:", acceptLangs)
		var langs []language.Tag
		for locale := range locales {
			tag, err := language.Parse(locale)
			if err != nil {
				return nil, fmt.Errorf("bad %v: %w", locale, err)
			}
			langs = append(langs, tag)
		}
		// fmt.Println("defined langs:", langs)
		matcher := language.NewMatcher(langs)
		tag, _, conf := matcher.Match(acceptLangs...)
		switch conf {
		case language.Exact:
		case language.High, language.Low:
			cfg.Logger.Infof("Using language %v", tag)
		case language.No:
			return nil, fmt.Errorf("no match for %q in recognized languages %v", cfg.Language, langs)
		}
		lang = tag
```